package.json

```diff
@@ -11,7 +11,7 @@
     "css-build": "./bin/build-css",
     "clean": "rm -rf public/* .port.tmp *.log build/* uploads/* test-results.xml",
     "lint": "eslint . --cache --max-warnings 0",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
running in a single thread is a lot faster than trying to run in parallel - between 1/3rd and a half faster